cmake/macrosForCreatingUnitTests.cmake

```diff
@@ -1,11 +1,11 @@
 
 # this function creates and adds an individual SERIAL unit-test
-macro(add_serial_utest TESTNAME TESTSRCS)
+macro(add_serial_utest TESTNAME)
   # set name of the executable
   set(testNameFinal ${TESTNAME})
 
   add_executable(${testNameFinal}
-    ${TESTSRCS} ${GTESTMAINSDIR}/gTestMain_serial.cc)
+    ${ARGN} ${GTESTMAINSDIR}/gTestMain_serial.cc)
```

Comment:
what changed here?

Reply:
`TESTSRCS` conveyed only first source file ignoring the rest: e.g. only `eigen_span.cc` was compiled into [`expressions_eigen_numerics`](https://github.com/Pressio/pressio/blob/tests_302/tests/functional_small/expressions/CMakeLists.txt#L6-L12)